### PR TITLE
docs(plugins/acme) add doc for dummy route setup and staging cleaning

### PR DESCRIPTION
**Pull request by fffonion**

### CODE REVIEW

File: app/_hub/kong-inc/acme/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ params:
       required: false
       default: "`https://acme-v02.api.letsencrypt.org`"
       description: |
-        The ACMEv2 API endpoint to use, user might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/) during testing.
+        The ACMEv2 API endpoint to use, user might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/) during testing. Kong doesn't automatically delete staging certificates, if you use same domain to test and use in production, you will need to delete those certificates manaully after test.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/acme/index.md]:
```suggestion
        The ACMEv2 API endpoint to use. You can use the [Let's Encrypt staging environment](https://letsencrypt.org/docs/staging-environment/) during testing. Note that Kong doesn't automatically delete staging certificates: if you use same domain to test and use in production, you will need to delete those certificates manually after testing.
```

File: app/_hub/kong-inc/acme/index.md [hunk 2]
```diff
@@ -115,7 +115,23 @@ verify Let's Encrypt API. The CA-bundle file is usually `/etc/ssl/certs/ca-certi
 Ubuntu/Debian and `/etc/ssl/certs/ca-bundle.crt` for CentOS/Fedora/RHEL.
 
 #### Enable the Plugin
+
+For all the domains that you need to get certificate, make sure `DOMAIN/.well-known/acme-challenge`
```
Comment (lena-larionova) [inline app/_hub/kong-inc/acme/index.md]: It sounds like you need to map this for each domain - if that's right, then use this phrasing:

```suggestion
For each the domain that needs a certificate, make sure `DOMAIN/.well-known/acme-challenge`
```

File: app/_hub/kong-inc/acme/index.md [hunk 3]
```diff
@@ -115,7 +115,23 @@ verify Let's Encrypt API. The CA-bundle file is usually `/etc/ssl/certs/ca-certi
...
+is mapped to a Route in Kong. You can check this by sending
+`curl KONG_IP/.well-known/acme-challenge/x -H "host:DOMAIN"` and expect a response `Not found`.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/acme/index.md]:
```suggestion
`curl KONG_IP/.well-known/acme-challenge/x -H "host:DOMAIN"` and getting the response `Not found`.
```

File: app/_hub/kong-inc/acme/index.md [hunk 4]
```diff
@@ -115,7 +115,23 @@ verify Let's Encrypt API. The CA-bundle file is usually `/etc/ssl/certs/ca-certi
...
+If not, add a Route and a dummy Service to catch this route.
 ```bash
+# add a dummy service if needed
+$ curl http://localhost:8001/service \
+        -d name=acme-dummy \
+        -d url=http://127.0.0.1:65535
+# add a dummy route if needed
```
Comment (lena-larionova) [inline app/_hub/kong-inc/acme/index.md]: Can you add a blank line right before this for consistency with the "add the plugin" section?

Comment (fffonion) [reply app/_hub/kong-inc/acme/index.md]: Added, thank you!

**Pull request merged**